input/pagecontent/changes.md

```diff
@@ -126,6 +126,7 @@ To help implementers, only the more significant changes are listed here.
   </li>
   <li>Removed codes from <a href="ValueSet-au-v3-ActEncounterCode-extended.html" >ActEncounterCode - AU Extended</a> (<a href="https://jira.hl7.org/browse/FHIR-47120">FHIR-47120</a>)
   </li>
+  <li>Deprecated <a href="StructureDefinition-encounter-description.html">Encounter Description</a> extension and removed reference to Encounter Description from <a href="StructureDefinition-au-encounter">AU Base Encounter</a> (<a href="https://jira.hl7.org/browse/FHIR-47121">FHIR-47121</a>)</li>
```

Comment:
This is two separate change log entries. The change log is structure by resource, not by a list of Jiras.

As this change introduces a second deprecation add a deprecation bullet point with two sub points, one for encounter description, and one for AU Base Diagnostic Request.

Put the change to the Encounter profile as a separate point (group with other changes to the encounter profile if they are present).